dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/search/SearchIncludeFragment.java

```diff
@@ -426,14 +463,13 @@ public String getSortOrder() {
      * method we could write a converter:
      * http://stackoverflow.com/questions/8609378/jsf-2-0-view-parameters-to-pass-objects
      */
-    public void setSortOrder(String sortOrderSupplied) {
-        if (sortOrderSupplied != null) {
-            if (sortOrderSupplied.equals(SortOrder.asc.toString())) {
-                this.sortOrder = SortOrder.asc;
-            }
-            if (sortOrderSupplied.equals(SortOrder.desc.toString())) {
-                this.sortOrder = SortOrder.desc;
-            }
+    public void setSortOrder(final String sortOrderSupplied) {
```

Comment:
Seems that it not change much. I would revert it, as it not add a value, and make tracking changes harder.